CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 -
 
+# v5.13.1
+
+- [changed] Upgraded Cloud Firestore client to v0.15.4.
```

Comment:
Please mention that you are now exporting the Timestamp class.

Reply:
Done